src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -358,35 +352,6 @@ function BaseReportActionContextMenu({
     );
 }
 
-export default withOnyx<BaseReportActionContextMenuProps, BaseReportActionContextMenuOnyxProps>({
-    betas: {
-        key: ONYXKEYS.BETAS,
-    },
-    reportActions: {
-        key: ({originalReportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`,
-        canEvict: false,
-    },
-    transaction: {
-        key: ({reportActions, reportActionID}) => {
-            const reportAction = reportActions?.[reportActionID];
-            return `${ONYXKEYS.COLLECTION.TRANSACTION}${(reportAction && ReportActionsUtils.getLinkedTransactionID(reportAction)) ?? -1}`;
-        },
-    },
-})(
-    memo(BaseReportActionContextMenu, (prevProps, nextProps) => {
-        const {reportActions: prevReportActions, ...prevPropsWithoutReportActions} = prevProps;
-        const {reportActions: nextReportActions, ...nextPropsWithoutReportActions} = nextProps;
-
-        const prevReportAction = prevReportActions?.[prevProps.reportActionID] ?? '';
-        const nextReportAction = nextReportActions?.[nextProps.reportActionID] ?? '';
-
-        // We only want to re-render when the report action that is attached to is changed
-        if (prevReportAction !== nextReportAction) {
-            return false;
-        }
-
```

Comment:
@pac-guerreiro I am having trouble with this place. It seems `lodashIsEqual(prevPropsWithoutReportActions, nextPropsWithoutReportActions);` can't detect the different between prevReportAction and nextReportAction

Reply:
I only want to make sure our change doesn't break any old behavior

Reply:
`reportActions` was injected into the component by `withOnyx` as a prop, but I removed `withOnyx` and `reportActions` is now returned by `useOnyx`, so it doesn't make sense to do this check anymore inside `memo`.

The component will be re-rendered if `useOnyx` is triggered by some change to `reportActions`, so I think the old behaviour should be the same